apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2.java

```diff
@@ -24,7 +25,8 @@
 import org.hl7.fhir.r4.model.Quantity;
 
 /** Transforms CCW {@link DMEClaim} instances into FHIR {@link ExplanationOfBenefit} resources. */
-final class DMEClaimTransformerV2 {
+public final class DMEClaimTransformerV2 {
```

Comment:
did the access on this class need to change to public?

Reply:
changed